File: pset1/more/mario.c
#include <cs50.h>
#include <stdio.h>

int n;

int main(void)
{
    do
    {
        printf("How many layers do you want to have for mario pyramids?  ");
        n = get_int();
    }
    while (n< 0 || n > 23);


    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j< n-i-1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j<i+1; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j<i+1; j++)
        {
            printf("#");
        }
        // for (int j = 0; j< n-i-1; j++)
        // {
        //     printf(" ");
        // }
        printf("\n");
    }
}

File: pset6/mario.py

def main():
    while True:
        print("How many layers do you want to have for mario pyramids?  ")
        n = int(input())
        if n>0 and n<23:
            break

    for i in range(n+1):
        print(""*(n-i), end='')        # print out blanks before any #
        print("#"*i, end='')           # print i number of #
        print(" "*2, end='')           # print two spaces in between
        print("#"*i)                   # print i number of # again


if __name__ == "__main__":
    main()